src/System.Runtime/tests/project.json

```diff
@@ -32,6 +32,11 @@
       "dependencies": {
         "System.Runtime.Serialization.Formatters": "4.3.0-beta-24530-03"
       }
+    },
+    "netcoreapp1.1": {
+      "dependencies": {
+        "System.Runtime.Serialization.Formatters": "4.3.0-beta-24530-03"
```

Comment:
I'm afk right now so can't remove the dependency and see what fails, but it was introduced in https://github.com/dotnet/corefx/commit/f6f97c206fa6b9edc48e1335cdc3d3d33cb54590#diff-29df68f37e76e464ad7ef9130076abae